aviatrix/utils.go

```diff
@@ -238,7 +231,7 @@ func sortSize(sizeList []string, i, j int) bool {
 	return false
 }
 
-// this function can sort the firewall_image_version format like: R81.10-335.883 && R81.10_rev1.0
+// versionFormat1 sorts the firewall_image_version format like: R81.10-335.883 && R81.10_rev1.0
```

Comment:
Should be `versionFormat1 compares...`

Reply:
Also the function name could be more descriptive. Maybe something like `compareCheckpointVersion`

Reply:
have fixed: compareCheckpointVersion compares firewall_image_version format like: R81.10-335.883 && R81.10_rev1.0